src/backend/core/product/migrations/0025_attributetype_value_type.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.0.3 on 2023-05-02 06:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("product", "0024_producttranslation_description_editorjs"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="attributetype",
+            name="value_type",
+            field=models.CharField(
+                choices=[
+                    ("TEXT", "Value is represented as a text"),
```

Comment:
do we need to distinguish between integer and decimal numbers? 
If not, we can create `number` value_type.

Reply:
Yes, see please end of this thread https://ecoseller-group.slack.com/archives/C04BC91HNDP/p1682853328725889